pennylane/operation.py

```diff
@@ -3064,10 +3064,11 @@ def enable_new_opmath_cm():
 
 
 # pylint: disable=too-many-branches
-def convert_to_legacy_H(op):
+def convert_to_H(op):
```

Comment:
Don't panic - `convert_to_legacy_H` is redefined below as a way to call this function with new_opmath temporarily disabled. Here the logic is generalized to make a version that will convert to a LinearCombination or Hamiltonian.

Reply:
Very happy with this change